examples/client.c

```diff
@@ -307,6 +307,22 @@ check_token(coap_pdu_t *received) {
     memcmp(received->token, the_token.s, the_token.length) == 0;
 }
 
+static int
+event_handler(coap_context_t *ctx,
+              coap_event_t event,
+              struct coap_session_t *session) {
+
+  switch(event) {
+  case COAP_EVENT_DTLS_CLOSED:
+  case COAP_EVENT_TCP_CLOSED:
+  case COAP_EVENT_SESSION_CLOSED:
+    quit = 1;
+    break;
+  default:
+    break;
+  }
+}
```

Comment:
I suggest adding

```c
  return 0;                     /* need to return a value */
```

to suppress the compiler warning.

Reply:
Fixed - No idea how I missed the compiler warning, but ....